### DISCUSSION TITLE
Can't delete error from main screen

### DISCUSSION THREAD

**Discussion by Frogi-x** [ente-io/ente#3743] [Q&A]
Hello I scanned different type code to check if ente will read this but no and What's worse, it added an entry on the main page where there are other codes "Error: We are unable to parse 1 code" and I can't delete it at all.

Please make update To be able to remove such errors or use themselves after 24 hours.

**Comment by jaggedjimmyj** [comment]
Bump! Encountered this issue too, and it is very annoying. Any ideas how to remove the error from my Ente screen?

**Comment by Aptul9** [comment]
after 2 years the issue is still here

**Comment by ua741** [comment]
Upcoming release will have an option to delete.
